docs/SalesforceMultiObjects-batchsource.md

```diff
@@ -12,14 +12,18 @@ and the value is the schema of the SObject.
 Configuration
 -------------
 
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
+
+**Connection:** Name of the connection to use.
+
 **Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
 
 **Username:** Salesforce username.
 
 **Password:** Salesforce password.
 
-**Security Token:** Salesforce security Token. If the password does not contain the security token the plugin 
-will append the token before authenticating with salesforce.
+**Security Token:** Salesforce security Token. If the password does not contain the security token, the plugin 
```

Comment:
change Salesforce security token to Salesforce security token

Reply:
done